test/e2e/allocation.new.test.js

```diff
@@ -10,7 +10,7 @@ test('Create allocation and verify the result', async t => {
   await t.navigateTo(`/allocation/${allocationId}`)
   ;['summary', 'meta'].forEach(async section => {
     for (const o of allocationJourney.allocationViewPage.nodes[section].keys) {
-      const textExists = allocationJourney.allocationViewPage.getDlDefinitionByKey(
+      const textExists = await allocationJourney.allocationViewPage.getDlDefinitionByKey(
```

Comment:
I pushed a similar change to fix this issue and just made it a bit more readable:
https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/564/commits/bc59ae218ea425f0ff0cc8e333a7783523c7545e

`o` in the loop seemed a bit ambiguous so updated it to be `key` and passed the promise directly into the expect call which is what we've done elsewhere.